manifests/eth.pp

```diff
@@ -93,6 +93,7 @@
 # @param net_type
 # @param netmask
 # @param network
+# @param nm_controlled
```

Comment:
This is a new feature and warrants a minor version bump

Reply:
:+1: